uk.co.compucorp.civicrm.tasksassignments/templates/CRM/Tasksassignments/Reminder/DailyReminderActivity.tpl

```diff
@@ -16,7 +16,7 @@
                     </td>
```

Comment:
Could you please fix the indentation in this file? If you need to both automatic and manual fixes - please do not mix them and commit in separate commits.